admin/src/containers/App.js

```diff
@@ -1,15 +1,14 @@
 import React, { Component } from 'react';
-import logo from 'assets/logo.png';
+import Routes from 'routes';
+import { ToastContainer } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
 
 class App extends Component {
   render() {
     return (
       <div className="App">
```

Comment:
@Jakedavies by doing this you wrapped the whole app in `div.App`, which changed the styling ;)

see on local:
![screen shot 2018-07-03 at 15 54 23](https://user-images.githubusercontent.com/1215868/42224006-67776e98-7ed9-11e8-9a55-54db872af8ff.png)

vs demo:
![screen shot 2018-07-03 at 15 54 31](https://user-images.githubusercontent.com/1215868/42224030-6fb6fbb4-7ed9-11e8-8148-f5742d888cc9.png)

I'll do a quick-fix on master :)